operations/boot_orchestration/BOS_Workflows.md

```diff
@@ -1,36 +1,361 @@
+
+1. **CFS applies configuration**
+
+    If needed, CFS runs Ansible on the nodes and applies post-boot configuration \(also called node personalization\).
+
+1. **Phase operator (complete)**
+
+    The phase operator will continue monitoring the states for each node until CFS reports that configuration is complete.
+    The phase operator will clear the current phase now that the node is in it's final state.  The phase operator will also disable components at this point.
+
+1. **Session completion operator**
+
+    When all nodes belonging to a session have been disabled, the session is marked complete, and it's final status is saved to the database.
+
+### V2 Power Off Nodes
+
+**Use Cases:** Administrator powers off selected compute nodes.
+
+**Components:** This workflow is based on the interaction of the Boot Orchestration Service \(BOS\) with other services during the node shutdown process:
+
+Mentioned in this workflow:
+
+* Boot Orchestration Service \(BOS\) is responsible for booting, configuring, and shutting down collections of nodes. The Boot Orchestration Service has the following components:
+  * Boot Orchestration Session Template is a collection of one or more boot set objects. A boot set defines a collection of nodes and the information about the boot artifacts and parameters.
+  * BOS sessions provide a way to apply a template across a group of nodes and monitor the progress of those nodes as they move toward their desired state.
+  * BOS Operators interact with other services to perform actions on nodes, moving them toward their desired state.
+* Cray Advanced Platform and Monitoring Control \(CAPMC\) service provides system-level power control for nodes in the system.
+CAPMC interfaces directly with the Redfish APIs to the controller infrastructure to effect power and environmental changes on the system.
+* Hardware State Manager \(HSM\) tracks the state of each node and their group and role associations.
+
+![Boot Nodes](../../img/operations/boot_orchestration/bos_v2_shutdown.png)
+
+**Workflow Overview:** The following sequence of steps occur during this workflow.
+
+1. **Administrator creates a session template**
+
+    A session template is a collection of metadata for a group of nodes and their desired configuration. A session template can be created from a JSON structure. It returns a Session Template ID if successful.
+
+    See [Manage a Session Template](Manage_a_Session_Template.md) for more information.
+
+1. **Administrator creates a session**
+
+    Create a session to perform the operation specified in the operation request parameter on the boot set defined in the session template. For this use case, Administrator creates a session with operation as Boot and specifies the session template ID.
+
+    ```bash
+    # cray bos v2 sessions create \
+    --template-name SESSIONTEMPLATE_NAME \
+    --operation Reboot
+    ```
+
+1. **Session setup operator**
```

Comment:
The session set-up operator isn't pulling anything from S3 in this case as it should just be setting the desired state to empty because we're shutting down.